{"name": "旧密码登录失败", "status": "broken", "statusDetails": {"message": "AttributeError: 'tuple' object has no attribute 'url'", "trace": "@allure.title(\"旧密码登录失败\")\n    def test_oldlogin():\n        url = 'http://qa.yansl.com:8084/login'\n        req = {\n            \"pwd\": \"123456zjcq\",\n            \"userName\":\"dru795\"\n        }\n>       resp = request_tool.post_json(url, json=req)\n\ntest_case\\User\\test_oldpwd.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ('http://qa.yansl.com:8084/login',)\nkwargs = {'json': {'pwd': '123456zjcq', 'userName': 'dru795'}}, r = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError"}, "attachments": [{"name": "log", "source": "4807578e-df28-4ee3-b1f4-7df850a93a90-attachment.txt", "type": "text/plain"}], "start": 1602684603642, "stop": 1602684608830, "uuid": "047797c4-2c96-4c28-babb-133daf5e1303", "historyId": "f9c7a95be7caf8c1653a90d1e70c8bd0", "fullName": "test_case.User.test_oldpwd#test_oldlogin", "labels": [{"name": "parentSuite", "value": "test_case.User"}, {"name": "suite", "value": "test_oldpwd"}, {"name": "subSuite"}, {"name": "host", "value": "DESKTOP-679Q8H2"}, {"name": "thread", "value": "15032-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.User.test_oldpwd"}]}